app/views/comments/index.html.erb

```diff
@@ -94,8 +94,7 @@
         <%= @commentable.title %>
       </h1>
       <div class="flex">
-        <a class="crayons-btn crayons-btn--outlined mr-2" href="<%= @commentable.path %>">View post</a>
-        <a class="view-discussion crayons-btn crayons-btn--outlined mr-2" href="<%= @commentable.path %><%= user_signed_in? ? "/" : "#" %>comments">Full discussion</a>
+        <a class="crayons-btn crayons-btn--outlined mr-2" href="<%= @commentable.path %>"><%= t("core.view") %> <%= t("core.post").downcase %></a>
```

Comment:
@benhalpern - Just having a wee peek at this PR, and in some places, I worry a bit about assumptions around grammar - e.g. View + post works for English, but in some languages, I'm pretty sure it's going to be more like "post + view". Same for things like "trending on" + "community name".

I know we can iterate, but I'm wondering if we need to make some tweaks around this to make sure we set the best practice for sentence/phrase construction from the outset?

Reply:
@aitchiss This and similar issues like date formats, pluralization, etc. have been discussed to a certain extent, I'll soon get started on some of those (probably date formats first).

Reply:
Cool 😎  I was just thinking we're likely to get a lot of contributions for localisation in October, and getting some guidance/best practice around this in preparation would be awesome. Sounds like you're all over it though 😄

Reply:
I just don't seem to have enough time for everything I want to do right now but I definitely want to at least get some examples for date formats and pluralization into the code base.